graphics/vtk/Portfile

```diff
@@ -122,13 +167,19 @@ variant hdf5 description {Add hdf5 readers} {
 }
 
 if {[mpi_variant_isset]} {
-    configure.args-append -DVTK_Group_MPI:BOOL=ON
+    configure.args-append \
+        -DVTK_Group_MPI:BOOL=ON \
+        -DVTK_USE_SYSTEM_DIY2:BOOL=OFF
+
 }
 
 if {![variant_isset python35] && ![variant_isset python36] && ![variant_isset python37]} {
```

Comment:
if you want to change the default variant to +python37, check in the line above should likely include ```python27``` instead of ```python37```

Reply:
You're right. Sorry :)